src/Sonarr.Api.V3/Qualities/QualityDefinitionController.cs

```diff
@@ -55,6 +55,12 @@ public object UpdateMany([FromBody] List<QualityDefinitionResource> resource)
                 .ToResource());
         }
 
+        [HttpGet("limits")]
+        public IActionResult GetLimits()
+        {
+            return Ok(_qualityDefinitionService.GetLimits());
```

Comment:
I don't think this is going to play nice with API docs.

Reply:
You mean swagger/OpenAPI? I wasn't aware you were using that, so I can make the necessary changes for that and verify the output is properly documented.

Having said that, if there's a specific pattern you'd like me to follow, I'd appreciate some clearer direction. I, of course, want to follow current standards required by the project.

Reply:
Yeah, they're automatically generated with a GHA.

Not sure if something like https://github.com/Sonarr/Sonarr/blob/develop/src/Sonarr.Api.V3/Series/SeriesImportController.cs#L19-L24 with a proper return type should do it. From the API typically we'd return a "Resource" so something like `QualityLimitsResource`

Reply:
I changed the return type to `ActionResult<QualityDefinitionLimits>` and I get this output when I generate docs from the exported Open API JSON file:
![image](https://github.com/user-attachments/assets/6252fd1d-9abf-4ac6-ba65-9fa5f7ba711d)

This will be available in my next push, with the rest of the changes.